airflow/cli/commands/connection_command.py

```diff
@@ -255,5 +255,4 @@ def _import_helper(file_path):
                 continue
 
             session.add(conn)
-            session.commit()
             print(f'Imported connection {conn_id}')
```

Comment:
Similarly, removing this `commit` will cause the `print` to happen too early, even if data commit does not actually succeed.